project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   val sentryRavenLogback = "com.getsentry.raven" % "raven-logback" % "7.2.3"
   val scalaUri = "com.netaporter" %% "scala-uri" % "0.4.6"
   val memsubCommonPlayAuth = "com.gu" %% "memsub-common-play-auth" % "0.7"
-  val membershipCommon = "com.gu" %% "membership-common" % "0.317"
+  val membershipCommon = "com.gu" %% "membership-common" % "0.320-SNAPSHOT"
```

Comment:
I was a bit confused! and yes, just fill in this number when you know it, then push that change.